app/models/button_maker.rb

```diff
@@ -19,7 +19,7 @@ class ButtonMaker
   def initialize(item, oclc)
     @item = item
     @oclc = oclc
-    @options = %w(call contact hold ill recall scan special_ill)
+    @options = %w(call contact hold recall ill scan special_ill)
```

Comment:
Does this control the order / priority of the buttons?

Reply:
yes! TIL :)

Reply:
Yup, and I'll add a comment accordingly.